future_challenges_for_RSEng.tex

```diff
@@ -350,26 +351,57 @@ \subsection{Development of the RSE community}
 \subsection{Impact of AI and automation}
 \label{sec:ai}
 \begin{whatis}{}{impact-ai-automation}
-RSEs understand how AI models work, and can help researchers reap their benefits, while
-clearly knowing their limits.
+RSEs understand how AI models work and how to tune and extend them.
+With these skills they can help researchers reap their benefits,
+while clearly knowing their limits.
 \end{whatis}
 Generative AI tools, such as GitHub Copilot~\cite{Friedmann2021},
-are being added to software development tools.
-These AI powered code assistants promise to increase productivity
+are being added to software development tools~\cite{Alenezi2025}.
+These AI-powered code assistants promise to increase productivity
 by automatically generating code, and writing test cases and documentation.
 Software developers use them to reduce the number of keystrokes,
-complete tasks and to recall syntax without having to consult the documentation~\cite{Liang2024}.
+focus on higher-level design rather than on idiosyncrasies,
+and recall syntax without having to consult the documentation~\cite{Liang2024}.
 Code assistants have the potential to help novice or even non-expert programmers
 write code~\cite{Feldman2024} thus enabling non-experts to participate in research
 and software creation, and fostering greater collaboration across disciplines.
 However, when these tools are used in practice, many users report increased cognitive
 load and frustration stemming from the difficulty to prompt the system effectively
 and from debugging the generated code~\cite{Simkute2025}.
+Many factors can influence AI tools adoption~\cite{Russo2024}.
 The potential of AI to be a democratising technology can only be realised
 when the algorithmic divide along with the digital divide is addressed~\cite{Yu2020}.
-
-Another risk of the widespread use of AI tools is that community resources
-such as \eg\ Stack Overflow and Wikipedia become poorer because there are fewer incentives to maintain them.
+We also note members of the SE community have called for improving LLM literacy
+by adapting SE curricula~\cite{Kirova2024}.
+
+Besides the direct impact in research software and further research output,
+we see a number of risks.
+The workshop participants suggested that the emergence of AI and automation might reduce critical thinking and imagination.
+We address this in \autoref{sec:complexity}.
+Another risk is that the widespread use of AI tools could make community resources
+such as \eg\ Stack Overflow and Wikipedia (which are used to train AI models) poorer
```

Comment:
I used conditional here to soften that claim. My two cents: there are a lot of hot takes on Stack Overflow right now, some like to blame AI, others like to blame pre-existing issues with the platform governance and/or community behavior. There are preprints on the effects of AI on community resources, and their conclusions are nuanced and multi-faceted.